include/CIL/RayTracing/Camera.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef CIL_CAMERA_HPP
+#define CIL_CAMERA_HPP
+
+#include <CIL/ImageMatrix.hpp>
+#include <CIL/RayTracing/Ray.hpp>
+
+namespace CIL {
+    class Camera
+    {
+      public:
+        Camera(const double viewport_width, const double viewport_height,
+               const double focal_length = 1.0,
+               const Point3D origin = Point3D(0, 0, 0))
+            : m_origin(origin), m_horizontal(Vector3D(viewport_width, 0, 0)),
+              m_vertical(Vector3D(0.0, viewport_height, 0.0))
+        {
+            m_origin = Point3D(0, 0, 0);
```

Comment:
It seems like this line was added initially for testing, and now should be removed.

Reply:
ah yes, leftover refactor